products/health-care/appointments/va-online-scheduling/express-care-research/express-care-conversation-guide.md

```diff
@@ -0,0 +1,111 @@
+
+Before we start, a few things I want to mention:
+
+- **This entire session should take about 45 minutes.** I want to be sure not to keep you much longer, so I may occasionally prompt you with the next question or topic.
+- **In this session, we want to hear your honest opinions.** We are not testing your ability. We just want to improve these tools to better meet Veteran's needs. I will not be offended by any opinions you express, and I welcome your feedback.
+- **You'll be interacting with a prototypes and a demo tool.** This is a demo tool that may not function exactly the way you expect. Some areas of the prototype will be clickable, and some will not. Since it's a demo, none of your actions will affect your actual VA information or benefits.
+- **If for any reason and at any time you want to stop the session, please let me know.** 
+- **Are you comfortable if I record my screen and audio as we talk today?** We use the recordings to confirm that we have captured your opinions accurately. The recordings are destroyed after we complete analysis, and none of your comments will be attributed to you directly. 
+  - If yes: **Once I start recording, I am going to confirm that you are ok with me recording this session once more.** 
+
+*Start recording.*
+
+- **I have started recording. I'd like to confirm: Are you comfortable if I record my screen the audio as we talk today?**
+
+## Section 0: Warm-up questions
+
+Before we look at the website, let’s start with a few warm-up questions.
+
+- Have you heard of "Express Care"?
+  - If YES: 
+    - What is it? 
+    - Have you used it?
+  - If NO: What do you think it might be?
+  - Prompt if needed: Is it Telehealth or in-person?
+- Have you ever used Telehealth appointments?
+  - If YES: 
+    - Can you tell us a bit more about your experience?
+  - If NO: 
+    - Have you considered it before? 
+    - What, if anything, is preventing you from trying it?
+- Are you aware of any forms of Telehealth appointments offered by VA providers?
+  - If YES: 
+    - Which ones?
+  - Have you ever utilized these services? 
+    - (Prompt if needed: VA or non-VA providers)
+  - How did you become aware of them?
+
+## Section 1: Value of VA urgent Telehealth
+
+- Have you heard anything about same-day/urgent care Telehealth appointments through VA?
+  - (Clarification question if needed: Do you think that VA offers any same-day urgent care appointments? Do you think that the VA offers any same-day urgent care Telehealth appointments?)
+- Have you ever tried to make a same-day appointment with a VA provider, either someone you see regularly or any available provider?
+  - If YES: 
+    - Can you tell me what happened?
+    - How did you make this appointment?
+    - Were you able to be seen?
+  - If NO:
+    - Have you ever considered doing this, but something prevented you? If so, what?
```

Comment:
Wonder if it's worth asking if they have ever done same-day appts with any of their other docs (non va) if applicable. ANd then if they have ever done same day telehealth appointments?

Reply:
Good catch -- I thought I had that in here. I have it in the other docs/had it in my original draft but moved things around and I lost it. I think it's worth asking for sure, especially since we are assuming part of the reason why they aren't using Express Care is bc they aren't aware of it as an offering from VA.

Reply:
oh, I see what I did actually. the second to last set of questions from section 0 are about using telehealth appts in general. I moved that question group to this section and added a few things in **bold** below:

- Have you ever used Telehealth appointments?
  - If YES: 
    **- Was this with a VA provider or non-VA doctor?**
        **- If VA: How did you schedule your appointment** 
    **- Was this a same-day appointment?**
    - Can you tell me a bit more about your experience?
  - If NO: 
    **- In your own words, can you define "Telehealth"?**
    - Have you considered it before? 
    **- What would convince you to try it?**
  **- Would you consider using Telehealth appointments for a same-day appointment?**